src/acceptance-test/java/org/zalando/nakadi/webservice/StorageControllerAT.java

```diff
@@ -10,6 +11,7 @@
 
 public class StorageControllerAT extends BaseAT {
 
+    @Ignore("Temporary ignore this test (zk_address should be passed as an env argument)")
```

Comment:
Wouldn't it be better to fix this test? Quite often, "temporary" becomes too easily "permanent". For example, we have a few tests in this project that have been "temporarily" skipped for several years now :)

Reply:
@lmontrieux yes, I think so too. For this test, I need to pass a zookeeper docker container address. I have 2 options:
- set up the ip address to some env variable and use it in the test
- add testcontainers support

What do you think?

Reply:
Although, I think testcontainers solves different issues (probably bring others), it should be a change not for single test, but to the project in general, and for that it would be nice to understand the advantages(disadvantages). Maybe you can file a ticket for that as well?

regarding ip address, isn't enough `nakadi.zookeeper.brokers` env var ?

Reply:
Did you know that we (testcontainers) have KafkaContainer, so that you don't have to deal with Docker Compose at all? ;)
https://github.com/testcontainers/testcontainers-java/blob/5d3904e1580f581dc42beaaaecd4f5ba3e5bc05d/modules/kafka/src/test/java/org/testcontainers/containers/KafkaContainerTest.java

But if you really want to use Docker Compose, there is DockerComposeContainer which solves your use case without any (known) issues :)

Just fyi :)

Reply:
Nice to see a main testcontainers contribute in our repo :)
Thank you for the info, yes, I did know it and it could be a way to go for us. This particular ticket was opened to solve one specific issue, and we moved to testcontainers in discussion. I suggested to open another ticket to clarify details for the whole team then it could be simpler for us to decide.

Reply:
@adyach feel free to mention / ping me in the conversation :)